cloud/aws/lambda-handler.py

```diff
@@ -2,27 +2,67 @@
 import subprocess
 import logging
 import base64
+from threading import Thread
 
-logging.basicConfig(level=logging.INFO)
+logging.getLogger().setLevel(logging.INFO)
+
+
+class ReturningThread(Thread):
+    """A wrapper around the Thread class to actually return the threaded function
+    return value when calling join()"""
+
+    def __init__(
+        self, group=None, target=None, name=None, args=(), kwargs={}, Verbose=None
```

Comment:
I'm probably just blind, but I can't spot where the `Verbose` argument is being used

Reply:
Copy pasted from Stackoverflow 😄. I am not very familiar with the python inheritance syntax and I assumed it was an override of the parent constructor, but I can see it's not!